ecc/bls12381/constants.go

```diff
@@ -9,34 +9,110 @@ type Scalar = ff.Scalar
 const ScalarSize = ff.ScalarSize
 
 var (
-	g1ParamB  = ff.Fp{ /*4*/ }  // 4
-	g1Param3B = ff.Fp{ /*12*/ } // 3*G1ParamB
-	g1GenX    = ff.Fp{
-		// 0xfb3af00adb22c6bb, 0x6c55e83ff97a1aef, 0xa14e3a3f171bac58,
-		// 0xc3688c4f9774b905, 0x2695638c4fa9ac0f, 0x17f1d3a73197d794,
+	g1Params struct{ B, _3B, GenX, GenY ff.Fp }
```

Comment:
having a private struct with public members is a bit odd

Reply:
done